test/models/test_rbm.py

```diff
@@ -63,7 +63,7 @@ def test_gcnn(use_bias, symmetries, lattice):
     g, hi, perms = _setup_symm(symmetries, N=3, lattice=lattice)
 
     ma = nk.models.GCNN(
-        symmetries=perms,
+        symmetry_info=perms,
```

Comment:
why is this called ```symmetry_info``` ? is there additional information passed rather than the permutations?

Reply:
Yes for the GCNN you need two arrays, an array of the permutation symmetries and either a product table or a list of irreducible representation matrices depending on how you are doing things. 

Right now GCNN allows you to pass a `Graph` or a `PermutationGroup` and the constructor will calculate the correct arrays (or you can just pass the arrays directly). 

I agree that we can do better than `symmetry_info` though!